components/gc-most-requested/gc-most-requested-en.html

```diff
@@ -53,7 +53,7 @@ <h2>Most requested</h2>
 
```

Comment:
Add the container to wrap the rest of the page's content. Make sure to add the closing tag. I cannot add it as a suggestion.
```suggestion
<div class="container">
```